cfme/fixtures/v2v_fixtures.py

```diff
@@ -206,7 +205,7 @@ def __vddk_url(): # noqa
     return url
 
 
-def __conversion_data(target_provider):  # noqa
+def get_conversion_data(target_provider):  # noqa
```

Comment:
You can remove `# noqa` here, I think it was only there because of the use of `__` in the function name before.